### PR TITLE
added my linkfree profile links and data

### PR DESCRIPTION
**Pull request by Nnadivictory25**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1142"></a>

### CODE REVIEW

**Review by KendallDoesCoding — APPROVED** [review]

File: public/data/Nnadivictory25.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Nnadi  Victory",
```
Comment (Panquesito7) [inline public/data/Nnadivictory25.json:2]: There's an extra space in your name. I recommend removing it. What do you think? 😄
```suggestion
    "name": "Nnadi Victory",
```

File: public/data/Nnadivictory25.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "milestones": [
+      {
+        "title": "Started Coding",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started freelancing"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/Nnadivictory25.json:27]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

File: public/data/Nnadivictory25.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Nnadi  Victory",
+    "type": "personal",
+    "bio": "Aspiring Fullstack Developer!",
```
Comment (Pradumnasaraf) [inline public/data/Nnadivictory25.json:4]:
```suggestion
    "bio": "Aspiring Full-stack Developer!",
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]

File: public/data/Nnadivictory25.json (hunk 2 above)
Comment (Nnadivictory25) [reply public/data/Nnadivictory25.json:27]: Thanks for this

File: public/data/Nnadivictory25.json (hunk 3 above)
Comment (Nnadivictory25) [reply public/data/Nnadivictory25.json:4]: Didn't see that coming 

Thanks mate

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]

**Pull request merged**